src/benchcab/benchcab.py

```diff
@@ -295,12 +305,18 @@ def fluxsite_run_tasks(self, config_path: str):
         tasks = self._get_fluxsite_tasks(config)
 
         logger.info("Running fluxsite tasks...")
+        logger.info(
+            f"tasks: {len(tasks)} ({self._fluxsite_show_task_composition(config)})"
+        )
         if config["fluxsite"]["multiprocess"]:
             ncpus = config["fluxsite"]["pbs"]["ncpus"]
             fluxsite.run_tasks_in_parallel(tasks, n_processes=ncpus)
         else:
             fluxsite.run_tasks(tasks)
-        logger.info("Successfully ran fluxsite tasks")
+
+        tasks_failed = [task for task in tasks if not task.is_done()]
+        n_failed, n_success = len(tasks_failed), len(tasks) - len(tasks_failed)
+        logger.info(f"{n_failed} failed, {n_success} passed")
```

Comment:
Would it be useful to users if there was a warning level log entry if there were any failed tasks? And some indication either of which tasks failed or how to find out which failed. This is valid for the bitwise comparison as well.

Reply:
I've checked that failure cases always output to the PBS log file. Here is the PBS log of a benchcab run with a task that fails due to a malformed namelist file:

```
/scratch/tm70/sb8430/conda/envs/benchcab-dev/bin/benchcab fluxsite-run-tasks --config=config.yaml
2024-05-20 12:30:37,479 - INFO - benchcab.benchcab.py:307 - Running fluxsite tasks...
2024-05-20 12:30:37,483 - INFO - benchcab.benchcab.py:308 - tasks: 20 (models: 1, sites: 5, science configurations: 4)
2024-05-20 12:30:37,615 - ERROR - fluxsite.fluxsite.py:252 - Error: CABLE returned an error for task AU-Tum_2002-2017_OzFlux_Met_R0_S0
2024-05-20 12:33:47,687 - INFO - benchcab.benchcab.py:319 - 1 failed, 19 passed
/scratch/tm70/sb8430/conda/envs/benchcab-dev/bin/benchcab fluxsite-bitwise-cmp --config=config.yaml
2024-05-20 12:33:48,725 - INFO - benchcab.benchcab.py:336 - Running comparison tasks...
2024-05-20 12:33:48,729 - INFO - benchcab.benchcab.py:337 - tasks: 0 (models: 1, sites: 5, science configurations: 4)
2024-05-20 12:33:48,773 - INFO - benchcab.benchcab.py:348 - 0 failed, 0 passed

======================================================================================
                  Resource Usage on 2024-05-20 12:33:52:
   Job Id:             116077189.gadi-pbs
   Project:            tm70
   Exit Status:        0
   Service Units:      1.93
   NCPUs Requested:    18                     NCPUs Used: 18              
                                           CPU Time Used: 00:40:46        
   Memory Requested:   30.0GB                Memory Used: 1.86GB          
   Walltime requested: 06:00:00            Walltime Used: 00:03:13        
   JobFS requested:    100.0MB                JobFS used: 0B              
======================================================================================

```